packages-content-model/roosterjs-content-model-editor/lib/editor/plugins/PastePlugin/WacComponents/processPastedContentWacComponents.ts

```diff
@@ -259,3 +263,13 @@ const wacListProcessor: ElementProcessor<HTMLOListElement | HTMLUListElement> =
         context.defaultElementProcessors.ul?.(group, element as HTMLUListElement, context);
     }
 };
+
+const wacBlockParser: FormatParser<ContentModelBlockFormat> = (
+    format: ContentModelBlockFormat,
+    element: HTMLElement
+) => {
+    if (element.classList.contains(TABLE_CONTAINER) && element.style.marginLeft.startsWith('-')) {
+        element.style.marginLeft = '0px';
```

Comment:
I'm not sure if we should change this margin value here. Normally a parser should not modify source element. 

Does it still work if we don't change the margin?

Reply:
You are right, I understand why this was not working. I should have been using the container formatParser instead of the block format parser. 

Just removed this line